source/contentRecog/__init__.py

```diff
@@ -59,6 +59,22 @@ def cancel(self):
 		"""
 		raise NotImplementedError
 
+	def validateCaptureBounds(self, left, top, width, height):

```

Comment:
Rather than putting the types in the doc string, the current code style we tend to use is type annotations.

```suggestion
	def validateCaptureBounds(self, left: int, top: int, width: int, height: int) -> bool:
```

I think ideally, this would handle a locationHelper.RectLTWH object as parameter instead, but I noticed that we use left, top, width and height as parameters all over the place in the module, so I think this is fine.

Reply:
>Rather than putting the types in the doc string, the current code style we tend to use is type annotations.

I just tried to mimic the docstrings used for other methods in the same class. Would it be ok to use a different style for only this method?

>I think ideally, this would handle a locationHelper.RectLTWH object as parameter instead, but I noticed that we use left, top, width and height as parameters all over the place in the module, so I think this is fine.

Yes, I only decided to use left, top, width and height specifically since the code already extracts these values from the `locationHelper.RectLTWH` object.

Reply:
> Would it be ok to use a different style for only this method?

In my opinion, yes. The other style was necessary before we used Python 3, as it didn't support type annotations. Now we're at Python 3 and it does support them, it is the ideal way to work as it also allows IDEs to validate the input you're throwing at these methods is correct.

Reply:
I think it would be cleaner to pass this as a `locationHelper.RectLTWH` object.